tasks/README.md

```diff
@@ -59,9 +59,13 @@ The preferred style of the keys should be `snake_case`.
 
 **parameters**:  The parameters or input the task accepts listed with a [Puppet data type](../language/types_values_variables.md) string and optional description. Top level params names have the same restrictions as Puppet class param names and must match `\A[a-z][a-z0-9_]*\z`. Parameters may be `sensitive` in which case the task runner should hide their values in UI where possible.
 
-**implementations**: A list of implementation files in preference order, along with an optional list of required features for that implementation to be suitable on a target. The available features are defined by the task runner, but task runners should define at least the `shell`, `powershell` and `puppet-agent` features.
+**files**: Array of file names specified as strings that include the module containing the file. Files must be saved in module directories that Puppet makes available via mount points: `files`, `lib`, `tasks`. File specifications ending with `/` will require the entire directory. 
```

Comment:
Do we prefer "array" or "list" for the description? I noticed for `implementations` "list" was used. In the JSON schema the type is "array".

Reply:
I don't believe JSON schema has a type that is "list". But we tend to use 'list' and 'array' interchangeably in description, and list is a bit more colloquial.